modules/library/src/test/scala/zio/elasticsearch/ElasticQuerySpec.scala

```diff
@@ -2027,7 +2048,15 @@ object ElasticQuerySpec extends ZIOSpecDefault {
           val queryWithNested         = nested(TestDocument.subDocumentList, nested("items", term("testField", "test")))
           val queryWithIgnoreUnmapped = nested(TestDocument.subDocumentList, matchAll).ignoreUnmappedTrue
           val queryWithInnerHits =
-            nested(TestDocument.subDocumentList, matchAll).innerHits(InnerHits().from(0).size(3).name("innerHitName"))
+            nested(TestDocument.subDocumentList, matchAll).innerHits(
+              InnerHits()
+                .from(0)
+                .size(3)
+                .name("innerHitName")
+                .highlights(highlight("stringField"))
```

Comment:
Maybe we should consider creating highlight that accepts `TestDocument.stringField` as parameter

Reply:
I would do that to a separate PR, if that is ok with you.